services/net/contentmigration/migrators/ClipMigrator.cs

```diff
@@ -64,18 +65,18 @@ public ClipMigrator(IApiService api, IOptionsSnapshot<MigratorOptions> migratorO
             // FilePath = (ingest.DestinationConnection?.GetConfigurationValue("path")?.MakeRelativePath() ?? "")
             //     .CombineWith($"{ingest.Source?.Code}/{GetDateTimeForTimeZone(ingest):yyyy-MM-dd}/", reference.Uid),
             Language = "", // TODO: Need to extract this from the ingest, or determine it after transcription.
-            //Tags = item.Categories.Select(c => new TNO.Kafka.Models.Tag(c.Name, c.Label))
-
         };
 
+        var auditUser = lookups.Users.FirstOrDefault(u => u.Username == this.Options.DefaultUserNameForAudit);
+
         // newsItem.string5 & newsItem.string5 seem to be the "Show/Program"
 
         if (newsItem.UpdatedOn != null)
         {
             content.UpdatedOn = newsItem.UpdatedOn != DateTime.MinValue ? newsItem.UpdatedOn.Value.ToUniversalTime() : null;
         }
 
-        if (newsItem.Tones != null)
+        if ((newsItem.Tones != null) && (newsItem.Tones.Any()))
```

Comment:
You can also do this `newsItem.Tones?.Any() == true`

Reply:
I like it. Much more succinct.